apps/dh/api-dh/source/DataHub.WebApi/Registration/DomainRegistrationExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Energinet.DataHub.Charges.Clients.Registration.Charges.ServiceCollectionExtensions;
+using Energinet.DataHub.MarketParticipant.Client.Extensions;
+using Energinet.DataHub.MessageArchive.Client.Extensions;
+using Energinet.DataHub.MeteringPoints.Client.Extensions;
+using Energinet.DataHub.Wholesale.Client;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Energinet.DataHub.WebApi.Registration
+{
+    public static class DomainRegistrationExtensions
+    {
+        public static void AddDomainClients(this IServiceCollection services, ApiClientSettings apiClientSettings)
```

Comment:
suggestion: "service collection extensions" should return the IServiceCollection to allow developers to chain methods.

Reply:
Thx. Fixed.